### ISSUE TITLE
Custom filters's date remains old

### ISSUE THREAD

**Issue by KKQ-KKQ** [AdguardTeam/AdGuardForSafari#348]
### Issue Details

I use this app for using only custom filters.
But ~~it~~ the date is not updated ~~actually~~ even when manually updating.
I enabled a verbose log but only success messages are recorded.
I opened a debug window and check verbose but nothing appeared when updating.

~~To update the filter, It needs deleting AdGuard's preferences and reinstall.~~

Log:
[2020-06-12 09:08:04.464] [info] On app activate
[2020-06-12 09:08:08.071] [info] Start checking filters updates..
[2020-06-12 09:08:08.071] [info] Checking updates for 5 filters
[2020-06-12 09:08:08.545] [info] Custom filters updated
[2020-06-12 09:08:08.546] [info] Filters updated successfully
[2020-06-12 09:08:09.622] [info] Starting loading filter rules from the storage
[2020-06-12 09:08:09.963] [info] Finished loading filter rules from the storage in 341 ms
[2020-06-12 09:08:09.963] [info] Starting request filter initialization.
[2020-06-12 09:08:09.964] [info] Finished request filter initialization in 1 ms. Rules count: 4056
[2020-06-12 09:08:10.466] [info] Loading content blocker.
[2020-06-12 09:08:10.466] [info] Rules loaded: 4056
[2020-06-12 09:08:10.473] [info] Affinity block general: rules length: 1
[2020-06-12 09:08:10.473] [info] Affinity block privacy: rules length: 1
[2020-06-12 09:08:10.473] [info] Affinity block security: rules length: 1
[2020-06-12 09:08:10.473] [info] Affinity block socialWidgetsAndAnnoyances: rules length: 1
[2020-06-12 09:08:10.474] [info] Affinity block other: rules length: 1
[2020-06-12 09:08:10.477] [info] Affinity block custom: rules length: 4055
* **Version of AdGuard for Safari:**
  * 1.7.4
* **Environment name and version:**
  * Safari 13.1.1
* **Operating system and version:**
  * macOS 10.15.5
* **Filters you use in AdGuard:**
  * Only One Custom Filter (https://280blocker.net/files/280blocker_adblock.txt)
* **Any other browser extensions you have installed:**
  * 1password

### Expected Behavior
Custom filters are updated correctly.

### Actual Behavior
Custom filters are not updated.
</details>

**Comment by KKQ-KKQ** [comment]
I might be wrong.
It seems that it updates but displayed date is old.
It might be a filter format issue.

**Comment by vozersky** [comment]
@zzebrum  pls confirm

**Comment by KKQ-KKQ** [comment]
I confirm with my Apache server.
The date remains old but the filter is updated.

I put the following into my filter.
```
[Adblock Plus 2.0]
! Title: TEST
! Homepage: http://www.localnet
! Licence: Unknown
! Last updated: 2020-05-31

!2020/05/31

|http://www.localnet/*
```
And Update it. (Disable a line)
```
[Adblock Plus 2.0]
! Title: TEST
! Homepage: http://www.localnet
! Licence: Unknown
! Last updated: 2020-06-18

!2020/06/18

!|http://www.localnet/*
```
I confirm that I cannot access my server at the first, and I can access at the second.
But Displayed date in AdGuard remains old.

**Comment by zzebrum** [comment]
yeap, I can confirm that.

**Comment by vozersky** [comment]
fixed in 1.8.3 beta
